pkgs/by-name/su/sunsetr/package.nix

```
@@ -6,21 +6,25 @@
 }:
 rustPlatform.buildRustPackage (finalAttrs: {
   pname = "sunsetr";
-  version = "0.7.1";
+  version = "0.7.4";
 
   src = fetchFromGitHub {
     owner = "psi4j";
     repo = "sunsetr";
     tag = "v${finalAttrs.version}";
-    hash = "sha256-XDa6kjhdEur8YDfQQNg+RpLRtfOeTklB6LwXJaPcG7c=";
+    hash = "sha256-lkYXEQkJ+QlK2WicZo9ZFm0Xezqk7FR/UM7AOrbK+p0=";
   };
 
-  cargoHash = "sha256-Jsii8PkRIZgQ4yrQHZpK8bLhaW5jg6EKYw65rPRCtGQ=";
+  cargoHash = "sha256-+2WGa5odKh7uqY/u5jOJVYGeBQ2j38mt4ODNjZDF3qc=";
 
   checkFlags = [
     "--skip=config::tests::test_geo_toml_exists_before_config_creation"
   ];
 
+  postInstall = ''
+    install -Dm755 $src/system-sleep/sunsetr-resume.sh $out/lib/systemd/system-sleep/sunsetr-resume
```

```suggestion
    install -Dm755 system-sleep/sunsetr-resume.sh $out/lib/systemd/system-sleep/sunsetr-resume
```
otherwise patches do not apply